sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -31,6 +31,8 @@ namespace detail {
 
 static constexpr int DbgProgMgr = 0;
 
+static const std::string UseSpvEnv{"SYCL_USE_KERNEL_SPV"};
```

Comment:
Why have global object instead of just constant string in the place where it's used?

Reply:
To avoid duplicating the constant string in the source file (albeit only once), replaced with a constexpr char[].